packages/babel-plugin-proposal-logical-assignment-operators/test/fixtures/logical-assignment/anonymous-function/input.js

```diff
@@ -0,0 +1,8 @@
+var a;
+console.log((a ||= function () {}).name);
+
+var b = 1;
+console.log((b &&= function () {}).name);
```

Comment:
Since this is just an input/output test, we don't need the "runtime logic". This should be enough:

```js
a ||= function () {};
a &&= function () {};
a ??= function () {};
```

Also, can you add:
1. Another test folder for arrow functions
1. Another test folder for named functions
1. One `exec.js` file in each of those three folders, to test the _runtime_ behaviors:

```js
var a, b = true, c;

a ||= function () {};
b &&= function () {};
c ??= function () {};

expect(a.name).toBe("");
expect(b.name).toBe("");
expect(c.name).toBe("");
```

Reply:
@nicolo-ribaudo Cool, I didn't know about the assertion type of tests. Let me update my PR.